jwst/pipeline/calwebb_coron3.py

```diff
@@ -142,18 +142,22 @@ def process(self, input):
 
         # Call the resample step to combine all the psf-subtracted target images
         result = self.resample(resample_input)
+        output_file = mk_prodname(self.output_dir, prod['name'], 'i2d')
```

Comment:
Someday this should be updated to use the library and/or datamodels functions that @stscieisenhamer implemented for use in other pipelines like calspec2 and calspec3, so that we can do away with standalone functions like mk_prodname. But that's not super important right now.

Reply:
This is coming in #1173.  Changes already made.  Just need to be tested and pushed to that PR.